backend/src/chats/user-chat/user-chat.controller.ts

```diff
@@ -34,7 +34,17 @@ export class UserChatController {
 	@Get(':id')
 	findOne(@Param('id') id: number) {
 		return this.userChatService.findOne({
-			where: { id },
+			/**
+			 *
+			 * THIS NEEDS TO CHANGE
+			 *
+			 */
+			where: { user_id: id },
+			/**
+			 *
+			 * THIS NEEDS TO CHANGE
+			 *
+			 */
```

Comment:
Ryno, this is something weird
when we were doing the PR where this code came into the main, it all worked, even though the UserChat entity does not have an `id` attribute. Still compiled/passed tests etc. 
When merging it with your branch it suddenly didn't work and complained that the UserEntity doesn't have an `id` attribute... So this is the quick fix for now and we'll look into it later... (maybe this whole controller file can be removed

Reply:
Noted!